src/Catel.MVVM/Catel.MVVM.Shared/Collections/Extensions/SuspensionContextExtensions.cs

```diff
@@ -76,6 +84,94 @@ public static ICollection<NotifyRangedCollectionChangedEventArgs> CreateEventArg
         }
 
         /// <summary>
+        /// The create mixed bash event args list.
+        /// </summary>
+        /// <param name="suspensionContext">The suspension context.</param>
+        /// <typeparam name="T">The type of collection item.</typeparam>
+        /// <returns>The <see cref="ICollection{NotifyRangedCollectionChangedEventArgs}"/>.</returns>
+        public static ICollection<NotifyRangedCollectionChangedEventArgs> CreateMixedBashEventArgsList<T>(this SuspensionContext<T> suspensionContext)
```

Comment:
If you want it public than please make the original function public but do not alter the behaviour.